jwcrypto/common.py

```diff
@@ -46,3 +46,57 @@ def __init__(self, message=None):
         if message:
             msg += ' (%s)' % message
         super(InvalidJWAAlgorithm, self).__init__(msg)
+
+
+class InvalidCEKeyLength(Exception):
+    """Invalid CEK Key Length.
+
+    This exception is raised when a Content Encryption Key does not match
+    the required lenght.
+    """
+
+    def __init__(self, expected, obtained):
+        msg = 'Expected key of length %d, got %d' % (expected, obtained)
```

Comment:
It's all; supposed to be bits, I'll change this error message too to make it clear